pom.xml

```diff
@@ -150,7 +150,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-slaves</artifactId>
-            <version>1.26</version>
+            <version>1.30.1</version>
```

Comment:
Can you use the BOM instead?

Reply:
do you mean to add a bom to the ssh-slaves plugin?

Reply:
No, to use `jenkinsci/bom` here rather than hard-coding individual plugin versions. Not necessary and could be done as a follow-up of course.

Reply:
lgtm, I will do it in another branch. Thanks